src/main.ts

```diff
@@ -51,6 +51,13 @@ const settingsSchema: SettingSchemaDesc[] = [
     description:
       "Size of the image to generate. Can be 256, 512, or 1024. Smaller images are faster to generate.",
   },
+  {
+    key: "shortcutBlock",
+    type: "string",
+    default: "mod+g",
```

Comment:
```suggestion
    default: "",
```
@briansunter Could you suggest another one? Or maybe simply remove the default. :)

Reply:
Hey @xxchan I still like the idea of having a default. I don't mind what the shortcut is, I don't know off the top of my head which ones are already being used by logseq. I think as long as it's not taken by the built in logseq shortcuts it should be fine.